src/corehost/build.sh

```diff
@@ -166,7 +166,7 @@ if [ -z $__commit_hash ]; then
 fi
 
 __build_arch_lowcase=$(echo "$__build_arch" | tr '[:upper:]' '[:lower:]')
-__base_rid=$__rid_plat-$__build_arch_lowcase
+export __base_rid=$__rid_plat-$__build_arch_lowcase
```

Comment:
Something like "Export __CrossToolChainTargetRID=$__base_rid" is what I would suggest.